### ISSUE TITLE
[critical] argument of type 'NoneType' is not iterable

### ISSUE THREAD

**Issue by chinskiy** [PaperMtn/slack-watchman#66]
Greetings!

**Describe the bug**
`slack-watchman` perfectly works with slack-watchman==4.3.0, but fails in 4.4.0 version with an error `[critical] argument of type 'NoneType' is not iterable`

**To Reproduce**
```
export SLACK_WATCHMAN_TOKEN=REDACTED
slack-watchman --timeframe w --all


        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⣀⣀⣀⡀⠀⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⠾⠛⢉⣉⣉⣉⡉⠛⠷⣦⣄⠀⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⠋⣠⣴⣿⣿⣿⣿⣿⡿⣿⣶⣌⠹⣷⡀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣼⠁⣴⣿⣿⣿⣿⣿⣿⣿⣿⣆⠉⠻⣧⠘⣷⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢰⡇⢰⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠀⠀⠈⠀⢹⡇⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⡇⢸⣿⠛⣿⣿⣿⣿⣿⣿⡿⠃⠀⠀⠀⠀⢸⡇⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⣷⠀⢿⡆⠈⠛⠻⠟⠛⠉⠀⠀⠀⠀⠀⠀⣾⠃⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠸⣧⡀⠻⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣼⠃⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢼⠿⣦⣄⠀⠀⠀⠀⠀⠀⠀⣀⣴⠟⠁⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⣦⠀⠀⠈⠉⠛⠓⠲⠶⠖⠚⠋⠉⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⣠⣾⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⠀⣠⣾⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⣾⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
        ⠀ ⠈⠛⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀

   Slack Watchman
   Slack enumeration and exposed secrets detection tool

   by PaperMtn - GNU General Public License

[>>] Slack Watchman started execution
[-] Version: 4.4.0
[-] Created by: PaperMtn <papermtn@protonmail.com>
[-] Searching workspace: REDACTED
[-] Workspace URL: https://REDACTED.slack.com/
[-] Downloading and importing signatures...
[-] Downloaded and processed signature file: archive_files.yaml
[-] Downloaded and processed signature file: budget_files.yaml
[-] Downloaded and processed signature file: excel_files.yaml
[-] Downloaded and processed signature file: powerpoint_files.yaml
[-] Downloaded and processed signature file: word_files.yaml
[-] Downloaded and processed signature file: bank_cards.yaml
[-] Downloaded and processed signature file: cusip_numbers.yaml
[-] Downloaded and processed signature file: cv_files.yaml
[-] Downloaded and processed signature file: date_of_birth.yaml
[-] Downloaded and processed signature file: drivers_licence_uk.yaml
[-] Downloaded and processed signature file: iban_numbers.yaml
[-] Downloaded and processed signature file: itin_numbers.yaml
[-] Downloaded and processed signature file: ni_numbers.yaml
[-] Downloaded and processed signature file: passport_numbers.yaml
[-] Downloaded and processed signature file: ssn_us.yaml
[-] Downloaded and processed signature file: certificates.yaml
[-] Downloaded and processed signature file: chef.yaml
[-] Downloaded and processed signature file: config_files.yaml
[-] Downloaded and processed signature file: dotfiles.yaml
[-] Downloaded and processed signature file: env.yaml
[-] Downloaded and processed signature file: executable_files.yaml
[-] Downloaded and processed signature file: filezilla.yaml
[-] Downloaded and processed signature file: http_password.yaml
[-] Downloaded and processed signature file: java.yaml
[-] Downloaded and processed signature file: jenkins.yaml
[-] Downloaded and processed signature file: kde_wallet.yaml
[-] Downloaded and processed signature file: macos.yaml
[-] Downloaded and processed signature file: openvpn.yaml
[-] Downloaded and processed signature file: password_safe.yaml
[-] Downloaded and processed signature file: pidgin_otr.yaml
[-] Downloaded and processed signature file: rdp.yaml
[-] Downloaded and processed signature file: s3_config.yaml
[-] Downloaded and processed signature file: sql.yaml
[-] Downloaded and processed signature file: ssh.yaml
[-] Downloaded and processed signature file: tunnelblick.yaml
[-] Downloaded and processed signature file: 1password_data_files.yaml
[-] Downloaded and processed signature file: akamai.yaml
[-] Downloaded and processed signature file: alibaba.yaml
[-] Downloaded and processed signature file: aws.yaml
[-] Downloaded and processed signature file: azure.yaml
[-] Downloaded and processed signature file: cloudflare.yaml
[-] Downloaded and processed signature file: facebook.yaml
[-] Downloaded and processed signature file: ftp.yaml
[-] Downloaded and processed signature file: generic_tokens.yaml
[-] Downloaded and processed signature file: github.yaml
[-] Downloaded and processed signature file: google.yaml
[-] Downloaded and processed signature file: grafana.yaml
[-] Downloaded and processed signature file: heroku.yaml
[-] Downloaded and processed signature file: mailchimp.yaml
[-] Downloaded and processed signature file: mailgun.yaml
[-] Downloaded and processed signature file: ms_teams.yaml
[-] Downloaded and processed signature file: new_relic.yaml
[-] Downloaded and processed signature file: openai.yaml
[-] Downloaded and processed signature file: pagerduty.yaml
[-] Downloaded and processed signature file: passwords.yaml
[-] Downloaded and processed signature file: paypal.yaml
[-] Downloaded and processed signature file: private_keys.yaml
[-] Downloaded and processed signature file: ruby.yaml
[-] Downloaded and processed signature file: shodan.yaml
[-] Downloaded and processed signature file: slack.yaml
[-] Downloaded and processed signature file: snyk.yaml
[-] Downloaded and processed signature file: stripe.yaml
[-] Downloaded and processed signature file: twilio.yaml
[-] Downloaded and processed signature file: twitter.yaml
[critical] argument of type 'NoneType' is not iterable
```

**Desktop (please complete the following information):**
```
python --version
Python 3.13.1

(venv) ➜ pip list
Package            Version
------------------ ----------
beautifulsoup4     4.12.3
certifi            2024.12.14
charset-normalizer 3.4.0
colorama           0.4.6
idna               3.10
pip                24.3.1
PyYAML             6.0.2
requests           2.32.3
slack-watchman     4.4.0
soupsieve          2.6
urllib3            2.2.3
```

**Comment by PaperMtn** [comment]
Hey @chinskiy, could you post the output you get when running with `—debug`?

**Comment by chinskiy** [comment]
Hello @PaperMtn , sure!

```
export SLACK_WATCHMAN_TOKEN=REDACTED
(test) ➜  test slack-watchman --timeframe w --all --debug


        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⣀⣀⣀⡀⠀⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⠾⠛⢉⣉⣉⣉⡉⠛⠷⣦⣄⠀⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⠋⣠⣴⣿⣿⣿⣿⣿⡿⣿⣶⣌⠹⣷⡀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣼⠁⣴⣿⣿⣿⣿⣿⣿⣿⣿⣆⠉⠻⣧⠘⣷⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢰⡇⢰⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠀⠀⠈⠀⢹⡇⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⡇⢸⣿⠛⣿⣿⣿⣿⣿⣿⡿⠃⠀⠀⠀⠀⢸⡇⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⣷⠀⢿⡆⠈⠛⠻⠟⠛⠉⠀⠀⠀⠀⠀⠀⣾⠃⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠸⣧⡀⠻⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣼⠃⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢼⠿⣦⣄⠀⠀⠀⠀⠀⠀⠀⣀⣴⠟⠁⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⣦⠀⠀⠈⠉⠛⠓⠲⠶⠖⠚⠋⠉⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⠀⠀⠀⣠⣾⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⠀⣠⣾⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
        ⠀⠀⠀⣾⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
        ⠀ ⠈⠛⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀

   Slack Watchman
   Slack enumeration and exposed secrets detection tool

   by PaperMtn - GNU General Public License

[>>] Slack Watchman started execution
[-] Version: 4.4.0
[-] Created by: PaperMtn <papermtn@protonmail.com>
[-] Searching workspace: REDACTED
[-] Workspace URL: https://REDACTED.slack.com/
[-] Downloading and importing signatures...
[#] Processing watchman-signatures-main/.github/workflows/testing.yml...
[#] Skipping unrecognized file: watchman-signatures-main/.github/workflows/testing.yml
[#] Processing watchman-signatures-main/.gitignore...
[#] Skipping unrecognized file: watchman-signatures-main/.gitignore
[#] Processing watchman-signatures-main/CHANGELOG.md...
[#] Skipping unrecognized file: watchman-signatures-main/CHANGELOG.md
[#] Processing watchman-signatures-main/LICENSE...
[#] Skipping unrecognized file: watchman-signatures-main/LICENSE
[#] Processing watchman-signatures-main/README.md...
[#] Skipping unrecognized file: watchman-signatures-main/README.md
[#] Processing watchman-signatures-main/models/__init__.py...
[#] Skipping unrecognized file: watchman-signatures-main/models/__init__.py
[#] Processing watchman-signatures-main/models/signature_basic.py...
[#] Skipping unrecognized file: watchman-signatures-main/models/signature_basic.py
[#] Processing watchman-signatures-main/models/signature_gitlab.py...
[#] Skipping unrecognized file: watchman-signatures-main/models/signature_gitlab.py
[#] Processing watchman-signatures-main/models/signature_slack_eg.py...
[#] Skipping unrecognized file: watchman-signatures-main/models/signature_slack_eg.py
[#] Processing watchman-signatures-main/models/signature_slack_std.py...
[#] Skipping unrecognized file: watchman-signatures-main/models/signature_slack_std.py
[#] Processing watchman-signatures-main/models/signature_stack_overflow.py...
[#] Skipping unrecognized file: watchman-signatures-main/models/signature_stack_overflow.py
[#] Processing watchman-signatures-main/requirements.txt...
[#] Skipping unrecognized file: watchman-signatures-main/requirements.txt
[#] Processing watchman-signatures-main/signatures/competitive/archive_files.yaml...
[-] Downloaded and processed signature file: archive_files.yaml
[#] Processing watchman-signatures-main/signatures/competitive/budget_files.yaml...
[-] Downloaded and processed signature file: budget_files.yaml
[#] Processing watchman-signatures-main/signatures/competitive/excel_files.yaml...
[-] Downloaded and processed signature file: excel_files.yaml
[#] Processing watchman-signatures-main/signatures/competitive/powerpoint_files.yaml...
[-] Downloaded and processed signature file: powerpoint_files.yaml
[#] Processing watchman-signatures-main/signatures/competitive/word_files.yaml...
[-] Downloaded and processed signature file: word_files.yaml
[#] Processing watchman-signatures-main/signatures/compliance/bank_cards.yaml...
[-] Downloaded and processed signature file: bank_cards.yaml
[#] Processing watchman-signatures-main/signatures/compliance/cusip_numbers.yaml...
[-] Downloaded and processed signature file: cusip_numbers.yaml
[#] Processing watchman-signatures-main/signatures/compliance/cv_files.yaml...
[-] Downloaded and processed signature file: cv_files.yaml
[#] Processing watchman-signatures-main/signatures/compliance/date_of_birth.yaml...
[-] Downloaded and processed signature file: date_of_birth.yaml
[#] Processing watchman-signatures-main/signatures/compliance/drivers_licence_uk.yaml...
[-] Downloaded and processed signature file: drivers_licence_uk.yaml
[#] Processing watchman-signatures-main/signatures/compliance/iban_numbers.yaml...
[-] Downloaded and processed signature file: iban_numbers.yaml
[#] Processing watchman-signatures-main/signatures/compliance/itin_numbers.yaml...
[-] Downloaded and processed signature file: itin_numbers.yaml
[#] Processing watchman-signatures-main/signatures/compliance/ni_numbers.yaml...
[-] Downloaded and processed signature file: ni_numbers.yaml
[#] Processing watchman-signatures-main/signatures/compliance/passport_numbers.yaml...
[-] Downloaded and processed signature file: passport_numbers.yaml
[#] Processing watchman-signatures-main/signatures/compliance/ssn_us.yaml...
[-] Downloaded and processed signature file: ssn_us.yaml
[#] Processing watchman-signatures-main/signatures/config_files/certificates.yaml...
[-] Downloaded and processed signature file: certificates.yaml
[#] Processing watchman-signatures-main/signatures/config_files/chef.yaml...
[-] Downloaded and processed signature file: chef.yaml
[#] Processing watchman-signatures-main/signatures/config_files/config_files.yaml...
[-] Downloaded and processed signature file: config_files.yaml
[#] Processing watchman-signatures-main/signatures/config_files/dotfiles.yaml...
[-] Downloaded and processed signature file: dotfiles.yaml
[#] Processing watchman-signatures-main/signatures/config_files/env.yaml...
[-] Downloaded and processed signature file: env.yaml
[#] Processing watchman-signatures-main/signatures/config_files/executable_files.yaml...
[-] Downloaded and processed signature file: executable_files.yaml
[#] Processing watchman-signatures-main/signatures/config_files/filezilla.yaml...
[-] Downloaded and processed signature file: filezilla.yaml
[#] Processing watchman-signatures-main/signatures/config_files/http_password.yaml...
[-] Downloaded and processed signature file: http_password.yaml
[#] Processing watchman-signatures-main/signatures/config_files/java.yaml...
[-] Downloaded and processed signature file: java.yaml
[#] Processing watchman-signatures-main/signatures/config_files/jenkins.yaml...
[-] Downloaded and processed signature file: jenkins.yaml
[#] Processing watchman-signatures-main/signatures/config_files/kde_wallet.yaml...
[-] Downloaded and processed signature file: kde_wallet.yaml
[#] Processing watchman-signatures-main/signatures/config_files/macos.yaml...
[-] Downloaded and processed signature file: macos.yaml
[#] Processing watchman-signatures-main/signatures/config_files/openvpn.yaml...
[-] Downloaded and processed signature file: openvpn.yaml
[#] Processing watchman-signatures-main/signatures/config_files/password_safe.yaml...
[-] Downloaded and processed signature file: password_safe.yaml
[#] Processing watchman-signatures-main/signatures/config_files/pidgin_otr.yaml...
[-] Downloaded and processed signature file: pidgin_otr.yaml
[#] Processing watchman-signatures-main/signatures/config_files/rdp.yaml...
[-] Downloaded and processed signature file: rdp.yaml
[#] Processing watchman-signatures-main/signatures/config_files/s3_config.yaml...
[-] Downloaded and processed signature file: s3_config.yaml
[#] Processing watchman-signatures-main/signatures/config_files/sql.yaml...
[-] Downloaded and processed signature file: sql.yaml
[#] Processing watchman-signatures-main/signatures/config_files/ssh.yaml...
[-] Downloaded and processed signature file: ssh.yaml
[#] Processing watchman-signatures-main/signatures/config_files/tunnelblick.yaml...
[-] Downloaded and processed signature file: tunnelblick.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/1password_data_files.yaml...
[-] Downloaded and processed signature file: 1password_data_files.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/akamai.yaml...
[-] Downloaded and processed signature file: akamai.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/alibaba.yaml...
[-] Downloaded and processed signature file: alibaba.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/aws.yaml...
[-] Downloaded and processed signature file: aws.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/azure.yaml...
[-] Downloaded and processed signature file: azure.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/cloudflare.yaml...
[-] Downloaded and processed signature file: cloudflare.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/facebook.yaml...
[-] Downloaded and processed signature file: facebook.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/ftp.yaml...
[-] Downloaded and processed signature file: ftp.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/generic_tokens.yaml...
[-] Downloaded and processed signature file: generic_tokens.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/github.yaml...
[-] Downloaded and processed signature file: github.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/google.yaml...
[-] Downloaded and processed signature file: google.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/grafana.yaml...
[-] Downloaded and processed signature file: grafana.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/heroku.yaml...
[-] Downloaded and processed signature file: heroku.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/mailchimp.yaml...
[-] Downloaded and processed signature file: mailchimp.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/mailgun.yaml...
[-] Downloaded and processed signature file: mailgun.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/ms_teams.yaml...
[-] Downloaded and processed signature file: ms_teams.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/new_relic.yaml...
[-] Downloaded and processed signature file: new_relic.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/openai.yaml...
[-] Downloaded and processed signature file: openai.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/pagerduty.yaml...
[-] Downloaded and processed signature file: pagerduty.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/passwords.yaml...
[-] Downloaded and processed signature file: passwords.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/paypal.yaml...
[-] Downloaded and processed signature file: paypal.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/private_keys.yaml...
[-] Downloaded and processed signature file: private_keys.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/ruby.yaml...
[-] Downloaded and processed signature file: ruby.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/shodan.yaml...
[-] Downloaded and processed signature file: shodan.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/slack.yaml...
[-] Downloaded and processed signature file: slack.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/snyk.yaml...
[-] Downloaded and processed signature file: snyk.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/stripe.yaml...
[-] Downloaded and processed signature file: stripe.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/twilio.yaml...
[-] Downloaded and processed signature file: twilio.yaml
[#] Processing watchman-signatures-main/signatures/tokens_and_credentials/twitter.yaml...
[-] Downloaded and processed signature file: twitter.yaml
[#] Processing watchman-signatures-main/tests/test_format_gitlab.py...
[#] Skipping unrecognized file: watchman-signatures-main/tests/test_format_gitlab.py
[#] Processing watchman-signatures-main/tests/test_format_slack_eg.py...
[#] Skipping unrecognized file: watchman-signatures-main/tests/test_format_slack_eg.py
[#] Processing watchman-signatures-main/tests/test_format_slack_std.py...
[#] Skipping unrecognized file: watchman-signatures-main/tests/test_format_slack_std.py
[#] Processing watchman-signatures-main/tests/test_format_stack_overflow.py...
[#] Skipping unrecognized file: watchman-signatures-main/tests/test_format_stack_overflow.py
[#] Processing watchman-signatures-main/tests/test_regex.py...
[#] Skipping unrecognized file: watchman-signatures-main/tests/test_regex.py
[critical] argument of type 'NoneType' is not iterable
[#] Traceback (most recent call last):
  File "/Users/chinskiy/Documents/test-watchman/lib/python3.13/site-packages/slack_watchman/__init__.py", line 300, in main
    signature_list = suppress_disabled_signatures(signature_list, auth_info.disabled_signatures)
  File "/Users/chinskiy/Documents/test-watchman/lib/python3.13/site-packages/slack_watchman/__init__.py", line 113, in suppress_disabled_signatures
    return [sig for sig in signatures if sig.id not in disabled_signatures]
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by PaperMtn** [comment]
Ok, it looks like this is an issue with the feature to suppress certain signatures, it looks like it fails if there is either no `watchman.conf` or there is one, but no signatures are added to be suppressed.

I will create a fix for this

**Comment by PaperMtn** [comment]
@chinskiy , this should now be fixed in version 4.4.1

**Comment by chinskiy** [comment]
Hello @PaperMtn,

Indeed, as you mentioned - I didn’t have a `watchman.conf` file.

Can confirm that the issue does not occur in 4.4.1

Thank you for the fast response and issue resolution!
